DetectorDescription/DetSensitive/src/SDWrapper.cpp

```diff
@@ -0,0 +1,35 @@
+#include "DDG4/Factories.h"
+#include "DD4hep/LCDD.h"
+
+#include "DetSensitive/SimpleTrackerSD.h"
+#include "DetSensitive/SimpleCalorimeterSD.h"
+
+namespace DD4hep {
+namespace Simulation {
+
+// Factory method to create an instance of SimpleTrackerSD
+static G4VSensitiveDetector* create_simple_tracker_sd(
```

Comment:
Ownership goes to the caller? Same question for the function further below. Just add a line to the comment.